src/lib/docs.ts

```diff
@@ -0,0 +1,27 @@
+import path from 'path'
+import { logInfoMessage, logError } from './logger.js'
+import Generator from '@asyncapi/generator'
+
+export default async (spec, config, resDir) => {
+  const configData = config.docs
+  if (configData?.enabled) {
```

Comment:
With this check, docs will only be generated if we add this configuration to the config file. It would be awesome if, by default, it generated docs and then people can disable it if they want to. So maybe it should be `configData?.disabled` instead? Folder can default to `docs` and template to `@asyncapi/markdown-template`.

Reply:
Why should it generate if it says `configData?.disabled` ? isn't that saying generate if disabled?

Reply:
shouldn't i be doing `!configData || configData?.enabled`?